app/views/scheduled_messages/_form.html.erb

```diff
@@ -1,7 +1,8 @@
 <%= simple_form_for @scheduled_message do |form| %>
   <%= form.input :title,
-    hint: "This is for identifying purposes only, the employee will not see the message title" %>
+    hint: "This is for identifying purposes only, the employee will not see the message title." %>
   <%= form.input :body, label: "Message body" %>
   <%= form.input :days_after_start, label: "Days after employee starts to send message" %>
+  <%= form.input :tag_list, label: "Tags", hint: "Separate tags with spaces to enter multiple tags at once." %>
```

Comment:
you changed this (from commas to spaces)! 

I think that comma separation is more typical in a case like this - what's the thinking behind spaces?